app/docs/config.json

```diff
@@ -0,0 +1,122 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos SDK - Legacy REST and gRPC Gateway docs",
```

Comment:
We can also update the swagger title to Crypto.org Chain?
```suggestion
    "title": "Crypto.org Chain - Legacy REST and gRPC Gateway docs",
```

Reply:
and this will be reflected on the generated swagger UI right?

Reply:
yeah, seems to be the case